src/components/Footer/Footer.js

```diff
@@ -1,9 +1,68 @@
+/* eslint-disable */
 import React from 'react';
+import { FaComment } from 'react-icons/fa';
+import { Link } from 'react-router-dom';
+import './Footer.scss';
 
 const Footer = () => {
   return (
-    <div>
-      <h1>footer</h1>
+    <div className="footer">
+      <div className="footerInner">
+        <div className="footerFirst">
```

Comment:
```suggestion
        <div className="linkBox">
```
그리고 해당 영역이 어떤 역할을 담당하고 있는 자식을 담고 있는지 className으로 담습니다.

이 영역은 어떤 텍스트를 누르면 해당 페이지로 이동하는 링크들을 담고 있는 영역이라
저라면 linkBox 라는 이름을 붙일 것 같네요!